doc/classes/Area2D.xml

```diff
@@ -51,7 +51,7 @@
 			<argument index="0" name="area" type="Node">
 			</argument>
 			<description>
-				If [code]true[/code], the given area overlaps the Area2D.
+				Returns [code]true[/code] if the given area intersects or overlaps Area2D. Otherwise it returns false.
```

Comment:
This can be shortened a bit:

```suggestion
				Returns [code]true[/code] if the given area intersects or overlaps Area2D, [code]false[/code] otherwise.
```